CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 1.2.0
+
+## Features and improvements
```

Comment:
nit: subtopics of the version should be H3 aka `###` . Could use an edit to v1.1.1 as well

Reply:
oof, great callout @asheemmamoowala but we've done this incorrectly for many releases. e.g. https://github.com/mapbox/mapbox-gl-js/releases?after=v0.50.0-beta.1 and https://github.com/mapbox/mapbox-gl-js/releases?after=v1.0.0